addons/binding/org.openhab.binding.onkyo/ESH-INF/thing/channel-groups.xml

```diff
@@ -0,0 +1,101 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="onkyo" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+    xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <!-- Channel Groups -->
+
+    <channel-group-type id="zone1Controls">
```

Comment:
It does not make sense to define the same channel group 3 times. There should be a single type "zoneControls", which is then used 3 times.

I see that you did not do this change, but maybe your could address this as well?